LIMS_IMAGE/web/templates/orders/results.html

```diff
@@ -1,37 +1,41 @@
 {% extends 'base.html' %}
 <!-- Not having this here makes the interpreter mad -->
+{% load static %}
 {% block title %}LIMS Home{% endblock %}
 <!-- Not having this here makes the interpreter mad -->
+{% block additionalStyles %}
+<link rel="stylesheet" href="{% static 'orders/css/results.css' %}" />
+<script src="{% static 'laboratory/js/sample_list_search.js' %}" defer></script>
+{% endblock %}
 <body>
-  {% block content%}
+  {% block content %}
   <div class="container">
     <h1>Results</h1>
-    <p>Your results are: </p>
-    
     {% if results %}
     <!-- results = {order_num: {test_sample_id: {status: string, result: string}}}} -->
-        <ul>
-            {% for order_num, samples in results.items %}
-            <li>Order Number: {{ order_num }}</li>
-            <ul>
-                {% for sample_test_id, test_result in samples.items %}
-                <li>Sample Id: {{ sample_test_id }}</li>
-                <ul>
-                    <li>Status: {{ test_result.status }}</li>
-                    {% if test_result.status == 'Completed' %}
-                    <li>Result: {{ test_result.result }}</li>
-                    {% endif %}
-                </ul>
+      <table>
+        <tr>
+          <th>Order Number</th>
+          <th>Sample ID</th>
+          <th>Status</th>
+          <th>Result</th>
+        </tr>
+        {% for order_num, samples in results.items %}
+          <tr>
+            <td>{{ order_num }}</td>
```

Comment:
Small Bug here:
![image](https://user-images.githubusercontent.com/35235493/160151098-26e18f48-f733-4604-b1ff-a15a900e1c02.png)
When I have multiple results for an order, they line up with other orders numbers, which makes it unclear what order this is for

Two different ways to fix this:
- https://www.w3schools.com/Tags/att_td_rowspan.asp (rowspan)
- https://www.geeksforgeeks.org/how-to-create-nest-tables-within-tables-in-html/ (nested tables)

You want each order number to have a row for each test_sample the is an entry for, like below
1003 | 
1013 | 1013-131-A-2
         | 1013-131-A-1
1004 |